skietbaan_fe/src/components/Login.js

```diff
@@ -158,7 +169,7 @@ class Login extends Component {
   }
 
   goToRegister() {
-    window.location = "/Register-page"
+    window.location = "/RegisterPage"
```

Comment:
camelCase please not CapitalCase

Reply:
why
its a url, it should all be lowercase

Reply:
https://wiredimpact.com/blog/never-use-capital-letters-urls/

Reply:
you guys